source/Octopus.Tentacle/Services/Scripts/Kubernetes/KubernetesScriptServiceV1Alpha.cs

```diff
@@ -169,14 +170,54 @@ static KubernetesScriptStatusResponseV1Alpha GetResponseForMissingScriptPod(Scri
                     new(ProcessOutputSource.StdErr, $"If you are using the default NFS storage, then also check if:"),
                     new(ProcessOutputSource.StdErr, $"- The NFS Pod was evicted due to exceeding its storage quota"),
                     new(ProcessOutputSource.StdErr, $"- The NFS Pod was restarted or moved as part of routine operation"),
-
-                }, 
+                },
                 lastLogSequence);
         }
 
         public bool IsRunningScript(ScriptTicket ticket)
         {
             return podStatusProvider.TryGetTrackedScriptPod(ticket) is not null;
         }
+
+        public async Task<KubernetesScriptStatusResponseV1Alpha> StartScriptAsync(StartKubernetesScriptCommandV1Alpha command, CancellationToken cancellationToken)
+            => (await StartScriptAsync(command.ToV1(), cancellationToken)).ToV1Alpha();
+
+        public async Task<KubernetesScriptStatusResponseV1Alpha> GetStatusAsync(KubernetesScriptStatusRequestV1Alpha request, CancellationToken cancellationToken)
+            => (await GetStatusAsync(new KubernetesScriptStatusRequestV1(request.ScriptTicket, request.LastLogSequence), cancellationToken)).ToV1Alpha();
+
+        public async Task<KubernetesScriptStatusResponseV1Alpha> CancelScriptAsync(CancelKubernetesScriptCommandV1Alpha command, CancellationToken cancellationToken)
+            => (await CancelScriptAsync(new CancelKubernetesScriptCommandV1(command.ScriptTicket, command.LastLogSequence), cancellationToken)).ToV1Alpha();
+
+        public Task CompleteScriptAsync(CompleteKubernetesScriptCommandV1Alpha command, CancellationToken cancellationToken)
+            => CompleteScriptAsync(new CompleteKubernetesScriptCommandV1(command.ScriptTicket), cancellationToken);
+    }
+
+    public static class CommandExtensions
```

Comment:
I'm thinking it would be worthwhile adding tests for these conversions - maybe a simple one to see that the properties are all copied across

Reply:
Sure

Reply:
Added in [`01a6c79` (#911)](https://github.com/OctopusDeploy/OctopusTentacle/pull/911/commits/01a6c793d7be8b596f62be8b49822d4b6799d728)